django/http/request.py

```diff
@@ -480,15 +480,22 @@ def build_request_repr(request, path_override=None, GET_override=None,
                else pformat(request.GET))
     except Exception:
         get = '<could not parse>'
+
     if request._post_parse_error:
         post = '<could not parse>'
     else:
         try:
-            post = (pformat(POST_override)
-                    if POST_override is not None
-                    else pformat(request.POST))
+            if POST_override is not None:
+                post = pformat(POST_override)
+            else:
+                # Use the exception reporter filter to return the cleansed POST data
+                from django.views.debug import get_exception_reporter_filter
```

Comment:
I see that changed happened since I started to work on this ticket. It seems that should have been done together in #12098, when the `build_request_repr` stopped beign used in `HttpRequest.__repr__`.

I'm not sure in what extend I should change the code, since `HttpRequest.__repr__` changed. It is definitely useless to print whole request object in every frame of the traceback in HTML, so that can be removed. Also I found there are 2 text representations of the error - one for AJAX requests and one in text version of error email. IMHO those two can be same as well. I'm working on a patch in this direction.